sdc/tests/test_series.py

```diff
@@ -4548,7 +4673,7 @@ def test_impl(s, axis):
             msg = 'Method cumsum(). Unsupported parameters. Given axis: int'
             self.assertIn(msg, str(raises.exception))
 
-    @skip_sdc_jit('Series.cov() parameter "min_periods" unsupported')
+    @unittest.skip("not work")
```

Comment:
Could you provide the reason here "why does it not work"?

Reply:
Fixed